src/components/not-found/not-found.tsx

```diff
@@ -0,0 +1,16 @@
+import { Link } from 'react-router-dom';
+import Header from '../header/header';
+import React from 'react';
+
+const NotFound = () : JSX.Element => (
+  <React.Fragment>
```

Comment:
Достаточно так написать, как ты написал обычно нужно только если key на фрагмент вешаем
```suggestion
  <>
```